src/python/pants/option/global_options.py

```diff
@@ -356,6 +359,10 @@ def register_bootstrap_options(cls, register):
             advanced=True,
             default=os.path.join(get_pants_cachedir(), "plugins"),
             help="Cache resolved plugin requirements here.",
+            removal_version="2.5.0.dev0",
```

Comment:
2.4.0

Reply:
Because it's become a noop, I'm not in much of a hurry on it.